app/parse.py

```diff
@@ -15,8 +25,85 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def parse_single_product(product_soup: BeautifulSoup) -> Product:
+    return Product(
+        title=product_soup.select_one(".title")["title"],
+        description=product_soup.select_one(
+            ".description"
+        ).text.replace("\xa0", " "),
+        price=float(product_soup.select_one(".price").text.replace("$", "")),
+        rating=int(len(product_soup.select("p .ws-icon.ws-icon-star"))),
+        num_of_reviews=int(
+            product_soup.select_one(
+                ".ratings > p.review-count"
+            ).text.split()[0]
+        ),
+    )
+
+
+def wright_products_to_csv(products: [Product], page: str) -> None:
```

Comment:
The function name `wright_products_to_csv` contains a typo. It should be `write_products_to_csv` to correctly reflect its purpose.